ci/infra/testrunner/tests/test_workers.py

```diff
@@ -8,3 +9,27 @@ def test_add_worker(bootstrap, skuba):
     workers = skuba.num_of_nodes("worker")
     assert masters == 1
     assert workers == 1
+
+
+def test_remove_worker(deployment, conf, platform, skuba, kubectl):
+    initial_workers = skuba.num_of_nodes("worker")
+    new_node_number = initial_workers
+
+    # Make sure we have at least one extra worker
+    assert platform.get_num_nodes("worker") > initial_workers
+
+    # Add an extra worker
```

Comment:
Same comment here regarding the need of adding one worker and then removing it. Why not start with a fully deployed cluster and remove one worker?